src/pages/components/split-button.mdx

```diff
@@ -102,22 +102,61 @@ import SplitButtonTypesCode from '!!raw-loader!../../examples/button/SplitButton
 
 ## API
 
+The SplitButton component follows this structure:
+
+```jsx
+<SplitButton>
+  <Button />
+  <Dropdown>
+    <Trigger>
+      <ChevronButton />
+    </Trigger>
+    <Menu>
+      <Item />
+      <Item />
+      {/* etc. */}
+    </Menu>
+  </Dropdown>
+</SplitButton>
+```
+
+<!-- markdownlint-disable no-emphasis-as-heading -->
+
+**Implementation notes**
+
+<!-- markdownlint-enable no-emphasis-as-heading -->
+
+The split button pattern is composed of:
```

Comment:
`The split button pattern contains:` perhaps?

Reply:
Updated!